src/Microsoft.Kusto.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -270,6 +269,24 @@ public virtual async Task<ConnectionCompleteParams> Connect(ConnectParams connec
             return completeParams;
         }
 
+        internal void RefreshAzureToken(string ownerUri)
```

Comment:
This calls back to ADS to get the User Token then updates the Connection and the Kusto Provider